lib/utils/request.js

```diff
@@ -206,6 +207,20 @@ function MULTIGET (fhurls, where, cb) {
   });
 }
 
+function _uploadHandlerFor(cb){
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Rename this '_uploadHandlerFor' function to match the regular expression ^[a-z][a-zA-Z0-9]*$ [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS100)